### ISSUE TITLE
CT - long distance flows aren't impacted by SWIM results

### ISSUE THREAD

**Issue by bettinardi** [tlumip/tlumip#64]
CT uses FAF flows which is good.  However when we do large, model shaking, scenarios like Rough Roads Ahead (RRA) the long distance flows and their destinations (like the Portland FAF zone) are static and completely non-reactive - "that's bad".  As we move into TLUMIP 6, we need to think of some way to "model" long distance flows so that they look like FAF (calibrated to FAF), but can react in some way.  This might be a non-starter, but I want to capture it so we can think about it and the implications as we move forward with TLUMIP.

**Comment by bettinardi** [comment]
I'm broadening this issue to do we need to re-develop CT in a larger way.  We have a series of open github issues around CT; #78 and #88 that demonstrate that CT needs additional clarity in outputs and operation, and recent comparison of SWIM results show larger fractures (quality issues) in the output from CT.  Of specific concern is the large and wide spread over assignment of truck vehicles on I-84.  The team will need to continue to evaluate what further effort towards improving CT looks like.

**Comment by bettinardi** [comment]
Wanted to capture related idea.  I think the team should investigate the CT flows at the external stations and better document how they are determined.  Since I-84 is consistently too high over the entire stretch (due to truck volumes), it occurs to me that the issue probably goes all the way back to the external station for I-84.  So it could be that we just need more peer review and documentation of the assumption that have been made to develop those incoming flows and then possibly adjusting I-84 down or shifting it to other external stations (maybe the flow comes in a different way).

This concept of understanding the external flows could also be used to address the original issue, "CT long distant flows aren't impacted by SWIM results".  If we know how the flows coming into SWIM are set, then we could have a system where a user could hand adjust them up or down depending on the scenario.  Or, instead of just using judgement, potentially the flow adjustment could be hooked to the NED feedback outcome, using those relationships to adjust the flows up and down.

So in summary, I think document and better exposing to the user how the external station truck flows are set could be the answer to the several strings in this issue.

**Comment by bettinardi** [comment]
Seems like the latest CT edits (Controls at the externals) will resolve this

**Comment by bettinardi** [comment]
resovled
